noir-projects/noir-protocol-circuits/crates/types/src/constants.nr

```diff
@@ -255,3 +255,4 @@ global GENERATOR_INDEX__PUBLIC_KEYS_HASH = 51;
 global GENERATOR_INDEX__NOTE_NULLIFIER = 52;
 global GENERATOR_INDEX__INNER_NOTE_HASH = 53;
 global GENERATOR_INDEX__NOTE_CONTENT_HASH = 54;
+global GENERATOR_INDEX__SYMMETRIC_KEY: u8 = 1;
```

Comment:
Why 1? This will collide with the note hash generator?

Reply:
Because we had 1 in deriveAESSecret function. But yes, it's retarded. Will update it to 54.

Reply:
Addressed in https://github.com/AztecProtocol/aztec-packages/pull/6020/commits/523cda1c4b7bb5f9da05dae8f87b9911822e29f7